packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -206,6 +214,94 @@ describe('Full migration flow for each project', { retries: { openMode: 2, runMo
     finishMigrationAndContinue()
   })
 
+  it('skips the file renaming migration-e2e-component-default-test-files', () => {
+    startMigrationFor('migration-e2e-component-default-test-files')
+    // default testFiles - auto
+    cy.get(renameAutoStep).should('exist')
```

Comment:
Also can you check the README for each project you added tests for and make sure it includes the right info? Each project has a readme, eg https://github.com/cypress-io/cypress/blob/10.0-release/system-tests/projects/migration-component-testing-defaults/README.md

It might be good to add a line like "this project has a default integration folder and default testFiles. We rename the integration folder, even if the user skips the spec rename".

My the question is, if they DO skip the renaming of test files, do we generate the correct `specPattern` in the config for them?

Reply:
I updated the component to include the custom pattern, and also the test to check to the valid config 
cypress.json
```json
{
  "componentFolder": "cypress/custom-component",
  "integrationFolder": "cypress/custom-integration"
}
```
cypress.config.js to be tested
```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  e2e: {
    setupNodeEvents (on, config) {
      return require('./cypress/plugins/index.js')(on, config)
    },
    specPattern: 'cypress/custom-integration/**/*.cy.{js,jsx,ts,tsx}',
  },
  component: {
    setupNodeEvents (on, config) {},
    specPattern: 'cypress/custom-component/**/*.cy.{js,jsx,ts,tsx}',
  },
})

```